subt/zmq-subt-x2.json

```diff
@@ -15,6 +15,9 @@
             "virtual_world": true
           }
       },
+      "localization": {
+        "driver": "subt.odoimuloc:Localization"
```

Comment:
did you test it? I guess that `init` is required parameter (or used to be). I would also add some "unit test" which can be later extended.

Reply:
Yes, I did test it:
<pre>0:09:36.386633 1 [[-4.05580510030742, 1.4821090662507161, 0.06753653943903415], [1.1361179273949959e-05, -5.319448228326778e-06, 0.9401963599595852, 0.3406329469698625]]
0:09:36.387090 27 [[-4.055806100307417, 1.482116066250727, 0.06778353943903404], [1.1361179273949959e-05, -5.319448228326778e-06, 0.9401963599595852, 0.3406329469698625]]
</pre>
It runs and returns reasonably similar results. 

I plan to add unittest - I just didn't want it the code to sit offline over the long weekend.

Reply:
OK, thanks

Reply:
One basic test is in. I'll add more hopefully later today.